rdr_service/model/bq_workbench_workspace.py

```diff
@@ -169,3 +172,31 @@ class BQRWBWorkspaceUsersView(BQView):
     __viewdescr__ = 'Research Workbench Workspace Users View'
     __pk_id__ = 'id'
     __table__ = BQRWBWorkspaceUsers
+
+
+class BQRWBAuditSchema(BQSchema):
```

Comment:
So we expect to bring over email and notes text unaltered, and move this to a more locked down PostgreSQL schema?

Reply:
The code to change the values to boolean for these fields is in the BQ generator.